Configurations/10-main.conf

```diff
@@ -1372,11 +1372,14 @@ sub vms_info {
         perlasm_scheme   => "auto",
         multilib         => "-x64",
     },
-    "VC-WIN32" => {
+    "VC-WIN64-ARM" => {
+        inherit_from     => [ "VC-WIN64-common" ],
+        cflags           => add("-D_ARM64_WINAPI_PARTITION_DESKTOP_SDK_AVAILABLE"),
+    },
+    "VC-WIN32-common" => {
         # x86 Win32 target defaults to ANSI API, if you want UNICODE,
         # configure with 'perl Configure VC-WIN32 -DUNICODE -D_UNICODE'
-        inherit_from     => [ "VC-noCE-common", asm("x86_asm"),
-                              sub { $disabled{shared} ? () : "uplink_common" } ],
+        inherit_from     => [ "VC-noCE-common" ],
         as               => sub { vc_win32_info()->{as} },
         asflags          => sub { vc_win32_info()->{asflags} },
         asoutflag        => sub { vc_win32_info()->{asoutflag} },
```

Comment:
The three `as*` attributes as well as `perlasm_scheme` should follow along to `VC-WIN32`.